lib/excon/instrumentors/logging_instrumentor.rb

```diff
@@ -0,0 +1,53 @@
+require 'logger'
+
+module Excon
+  class LoggingInstrumentor
```

Comment:
I wonder a bit if we should iterate on the existing [standard instrumentor](https://github.com/excon/excon/blob/master/lib/excon/standard_instrumentor.rb) to give better behavior around different usage and more flexibility on where things are output, rather than creating a new instrumentor with some overlap? What do you think?

Reply:
I thought about that, but wasn't sure if you would object to messing with it. I'd be happy to rework this PR if you prefer that approach.

Reply:
I'm open to this, as long as we can do so in a way that is additive (I wouldn't want to break backwards compatibility). Does that make sense?

Reply:
That might be tricky, but I don't know how many people are relying on $stdout output in production code. Seems like it's purely for debugging, which maybe it should stay, and we should stick with this approach, i.e. separate instrumentors.

Reply:
Yeah, it's hard to say for sure. Maybe if default logger is still stdout we can make it work, but maybe not. As you suggest it may be more trouble than it's really worth.